.gitignore

```diff
@@ -544,4 +544,6 @@ source/.editorconfig
 # Rider IDE
 source/.idea
 !*ClientDebug.csproj.user
-!*MissionTestMod.csproj.user
\ No newline at end of file
+!*MissionTestMod.csproj.user
+doc/BannerlordCoopConnectionStates.png
+doc/Architecture.png
```

Comment:
These have an issue with line endings, should be fixed on main.

https://stackoverflow.com/questions/1575682/cant-seem-to-discard-changes-in-git

```suggestion
```